### PR TITLE
Correct console log message to react-native eject

### PR DESCRIPTION
**Pull request by arnabsen**
react-native android is not a valid command. react-native eject will recreate the missing native folder.

## Test Plan

1. Delete android folder in project root
2. In console enter "react-native run-android"
3. Error message will now give hint to rebuild missing android folder via "react-native eject" command rather than non-existent "react-native android" command

## Release Notes

 [CLI] [BUGFIX] [local-cli/runAndroid/runAndroid.js] - Correct console log message to react-native eject

### CODE REVIEW

**Comment by arnabsen** [comment]
This is ready for review. Is there a way to trigger circle-ci changes without another commit?

**Comment by arnabsen** [comment]
Hi @facebook-github-bot I have rebased. The change is still valid, affecting this line- https://github.com/facebook/react-native/blob/master/local-cli/runAndroid/runAndroid.js#L37

**Comment by arnabsen** [comment]
Hi @ide mind having a quick look at this PR?

**Comment by arnabsen** [comment]
Hi @hramos mind having a quick look at this PR?

**Comment by arnabsen** [comment]
Hi @samuela mind having a quick look at this PR?

File: local-cli/runAndroid/runAndroid.js [hunk 1]
```diff
@@ -39,7 +39,7 @@ function runAndroid(argv, config, args) {
     } else {
       console.log(
         chalk.red(
-          'Android project not found. Maybe run react-native android first?',
+          'Android project not found. Maybe run react-native eject first?',
```
Comment (samuela) [inline local-cli/runAndroid/runAndroid.js]: nit: maybe \`react-native eject\` in some sort of quotes in order to clarify exactly what the command is?

**Comment by arnabsen** [comment]
Thanks @samuela, I added the quotes

**Comment by hramos** [comment]
In which situation would someone run into this? I understand `react-native eject` is only useful if you create a new project using CRNA. Is there any possibility someone might run into this when they are not using CRNA?

**Comment by arnabsen** [comment]
Hi @hramos if the android folder was accidentally deleted for example while trying to debug :no_mouth:

**Comment by arnabsen** [comment]
Hello @samuela can this PR be merged?

**Comment by samuela** [comment]
@arnabsen I don't have merge privileges unfortunately.

**Comment by arnabsen** [comment]
Hi @hramos which command do you suggest to restore a corrupt/missing/deleted android folder?

**Comment by cpojer** [comment]
Thank you for your PR. We just moved this code to https://github.com/react-native-community/react-native-cli so please submit your PR there.

**Pull request closed without merging**